eclipse-theia/theia
# stream webview resources to avoid blocking a web socket

#### What it does


- fix #8358: stream webview resources to avoid blocking a web socket

#### How to test


- install python extension from open vsx registry
- call `Create New Blank Jupyter Notebook`
- UI should not be blocked, the notebook should be loaded promptly


#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
